### PR TITLE
Forms Automated Test - LPS-132461 | maste

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase [hunk 1]
```diff
@@ -120,13 +120,67 @@ definition {
...
+			fieldName = "Grid",
+			fieldValue = "Grid Help Text");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.assertHelpTextPresent(
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase:167]: @evanilson-santana Is it possible to use `FormFields.viewFieldHelpText` instead?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase [hunk 2]
```diff
@@ -120,13 +120,67 @@ definition {
...
+			rowNumber = "1",
+			optionValue = "Option 2");
+
+		
+		FormFields.editHelpText(
+			fieldName = "tip",
+			key_fieldLabel = "",
+			fieldHelpText = "Grid Help Text");
+
+		FormViewBuilder.assertHelpTextPresent(
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase:157]: @evanilson-santana Is it possible to use` FormFields.viewFieldHelpText` instead?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase [hunk 3]
```diff
@@ -120,13 +120,67 @@ definition {
...
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Grid");
+
+		var gridFieldName = FormFields.getFieldName();
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase:140]: @evanilson-santana Do you need this variable? I noticed you're not using it.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase [hunk 4]
```diff
@@ -120,13 +120,67 @@ definition {
...
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Grid",
+			fieldValue = "Grid Help Text");
+		
+		
+		FormFields.selectGridRadioOptionWithoutCategory(
+			optionNumber1 = "1",
+			optionNumber2 = "2",
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase]: @evanilson-santana How about renaming to `optionRowNumber` and `optionColumnNumber`, respectively?

**Comment by evanilson-santana** [comment]
Fixed.

**Comment by carolmariaabb** [comment]
@evanilson-santana please fix the conflicts. Thanks

**Comment by evanilson-santana** [comment]
Fixed.

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**